test/jasmine/tests/gl_plot_interact_test.js

```diff
@@ -0,0 +1,55 @@
+var d3 = require('d3');
+
+var Plotly = require('@src/index');
+
+/*
+ * The following test cases fail on the CircleCI
+ * most likely due to a WebGL/driver issue
```

Comment:
> Is this the same issue with the bunny-hill test?

No. We can't have WebGl karma tests at the moment because karma fails to render `<canvas>` on the CircleCI box.

I'm not sure what's special about the bunny hull test case that makes output random pixel in 1 out of 10 runs. Maybe increasing the [delay](https://github.com/plotly/plotly.js/blob/master/src/snapshot/index.js#L15) on snapshot can fix it.